service-di/src/main/java/com/wednesday/template/service/ServiceModule.kt

```diff
@@ -17,13 +19,18 @@ val serviceModule = module {
 
     single { getOpenWeatherRetrofit(get(), get<OpenWeatherApiKeyInterceptor>()) }
 
+    single { getLastFMRetrofit(get()) }
+
```

Comment:
Both these types might be the same (Retrofit). You may need to use a named registeration